1000_Oe_field_scan_sequence.seq

```diff
@@ -0,0 +1,11 @@
+FLD FIELD -1000.0 20.0 0 0
+TMP TEMP 5.000000 3.000000 0
+WAI WAITFOR 60 1 1 0 0 0
+LPB SCANH -1000.0 0.0 20.0 51 0 3 0
+RES 63 25 2 50.000 500.000 0 0 95.0 0 100.000 10.000 0 0 10.0 0 100.000 10.000 0 0 10.0 0 100.000 10.000 0 0 10.0
+ENB EOS
+LPB SCANH 0.0 1000.0 20.0 51 0 3 0
+RES 63 25 2 50.000 500.000 0 0 95.0 0 100.000 10.000 0 0 10.0 0 100.000 10.000 0 0 10.0 0 100.000 10.000 0 0 10.0
+ENB EOS
+FLD FIELD 0.0 20.0 0 0
+TMP TEMP 300.000000 2.000000 0

```

Comment:
Last sweep should be oscillating not linear. Oscillating cancels residual magnetic field by reversing a few times (overshoots and then settltes).